py/torch_tensorrt/dynamo/runtime/_MutableTorchTensorRTModule.py

```diff
@@ -475,7 +475,9 @@ def _process_kwarg_inputs(inputs: Any) -> Any:
             + "Allowed input types: {torch_tensorrt.Input, torch.Tensor, list, tuple, dict}"
         )
 
-    def forward(self, *args: Any, **kwargs: Any) -> Any:
+    # Due to https://github.com/pytorch/pytorch/issues/157183, we cannot use forward as a workaround.
+    # This is a temporary fix.
+    def _forward(self, *args: Any, **kwargs: Any) -> Any:
```

Comment:
Leave the original forward, you can throw a warning or something but we should not break existing apis 

```py
def forward(self, *args: Any, **kwargs: Any) -> Any:
    warnings.warn("Direct calls to {self.__class__}.forward() are currently broken by due to https://github.com/pytorch/pytorch/issues/157183. Either call {self.__class__}(...) directly or use {self.__class__}._forward as a work around")
    return self._forward(*args, **kwargs)

```